plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1033,8 +1184,7 @@ private boolean isMaterializedView(Table table)
     @Override
     public List<SchemaTableName> listMaterializedViews(ConnectorSession session, Optional<String> schemaName)
     {
-        // Iceberg does not support VIEWs
-        // Filter on ICEBERG_MATERIALIZED_VIEW_COMMENT is used to avoid listing hive views in case of a shared HMS
+        // Filter on ICEBERG_MATERIALIZED_VIEW_COMMENT is used to avoid listing hive views in case of a shared HMS and to distinguish from standard views
```

Comment:
Sadly, ICEBERG_MATERIALIZED_VIEW_COMMENT is not Iceberg-specific (`"Presto Materialized View"`). so doesn't look right. cc @raunaqmorarka 

preexisting, no change requested here in this PR